mlserver/cli/serve.py

```diff
@@ -23,7 +23,7 @@ async def load_settings(
     if folder:
         sys.path.insert(0, folder)
 
-    settings = None
+    settings: Settings
```

Comment:
Is this valid as-is? I would add back the `= None` for explicitness (i.e. `settings: Optional[Settings] = None`).